server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1820,7 +1820,7 @@ private RepositoryData getRepositoryData(long indexGen) {
             return RepositoryData.EMPTY;
         }
         try {
-            final String snapshotsIndexBlobName = INDEX_FILE_PREFIX + Long.toString(indexGen);
+            final String snapshotsIndexBlobName = INDEX_FILE_PREFIX + indexGen;
```

Comment:
I'm curious whether underneath it generates the same code? Do you know?

Reply:
Hi @dblock , most probably, yes, I've tested that out.
And thanks for merging this.